hdd_tools/translations/en.yaml

```diff
@@ -35,4 +35,7 @@ configuration:
   output_file:
     name: Output file
     description: Log file
- 
\ No newline at end of file
+  device_type:
+    name: Device type
+    description: some block device types require to be explicityly declared for smartctl to work, for example some USB bridges (https://www.smartmontools.org/wiki/USB)
```

Comment:
Move this too after `hdd_path`.

All the descriptions in the file start with uppercase and one little typo (explicityly -> explicitly) :

But maybe is better to simply copy&paste the same text that you added to the official documentation, it seems ok to me:
> Type of block device which `smartctl` will try to use for communication. Default "auto". Check https://www.smartmontools.org/wiki/USB